Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.5.7-alpine3.10
+RUN apk add git
+WORKDIR /opt/catpol
+COPY ./requirements* /opt/catpol/
+RUN pip3 install -r requirements-dev.txt
+COPY . /opt/catpol
+ENV DJANGO_SETTINGS_MODULE=project_template.settings.dev
```

Comment:
Change this to `ENV DJANGO_SETTINGS_MODULE=project_template.settings.production`

Reply:
Changed in https://github.com/code4romania/catpol-declaratii/pull/177/commits/8c3692c414895e7ea863d09f79cd98f6ff5878ea